.github/workflows/publish-pypi.yml

```diff
@@ -0,0 +1,20 @@
+name: publish-PyPI CD
+
+on:
```

Comment:
Since we'd like to test this change before merging, you can temporarily add an extra:
```
  pull_request:
    branches: [ master ]
```
section here. Once we have confirmed that the change works as expected, we can remove this part and merge the PR.